scripts/openvpn/makeOVPN.sh

```diff
@@ -96,8 +96,21 @@ do
             if command -v bw > /dev/null; then
                 BITWARDEN="2"
             else
-               echo "Bitwarden not found, please install bitwarden"
-               exit 1
+               echo 'Bitwarden not found, please install bitwarden'
+
+				if grep -qsEe "^NAME\=['\"]?Alpine[a-zA-Z ]*['\"]?$" /etc/os-release
+				then
+					echo 'You can download it through the following commands:'
+					echo $'\t' 'curl -fLo bitwarden.zip --no-cache https://github.com/bitwarden/clients/releases/download/cli-v2022.6.2/bw-linux-2022.6.2.zip'
+					echo $'\t' 'apk --no-cache -X http://dl-cdn.alpinelinux.org/alpine/edge/testing/ add atool'
```

Comment:
Or does `apk` have issues with HTTPS?
```suggestion
					echo $'\t' 'apk --no-cache -X https://dl-cdn.alpinelinux.org/alpine/edge/testing/ add atool'
```

Reply:
This is the official repo of Alpine, so ...